services/webhooks/service.go

```diff
@@ -105,9 +105,10 @@ func (s *service) newCallFromResponse(requestTrace []byte, response *http.Respon
 		return nil, err
 	}
 
-	if body != nil {
-		w.Response = append(w.Response, body...)
-		w.ResponseBody = body
+	w.Response = append(w.Response, body...)
```

Comment:
even if we ignore a non-JSON body, it should still appear in the response trace of event